MobileWallet/Screens/Send/AddNote/AddNoteViewController.swift

```diff
@@ -190,6 +190,7 @@ class AddNoteViewController: UIViewController, UITextViewDelegate, SlideViewDele
         //Init first so it starts listening for a callback right away
         let sendingVC = SendingTariViewController()
         sendingVC.tariAmount = amount
+        sendingVC.recipientPubKey = recipientPubKey
         sendingVC.startListeningForDiscovery()
```

Comment:
Imo it would look better if you had the contruction & transition logic in a separate method, it might be a bit too much responsibility for `slideViewDidFinish`.